docs/General/loginToRPS.md

```diff
@@ -1,17 +1,17 @@
 
-The web portal is available for login after the deployment of [Management Presence Server (MPS)](../Glossary.md#m) and [Remote Provisioning Server (RPS)](../Glossary.md#r). Make sure both are running before attempting to login.
+The web portal is available for login after the deployment of the [Management Presence Server (MPS)](../Glossary.md#m), [Remote Provisioning Server (RPS)](../Glossary.md#r), and [Sample Web UI](../Glossary.md#s). Make sure both are running before attempting to login.
```

Comment:
I hate asking this as I don't like having debates in this tool. It's horrendous. But does this make sense now given where we are going with the one build instruction routine? Yes, I think it does to answer my own question. I just wonder if it will to the user. Let's definitely test this in the DXWT.